aviary/utils/process_input_decks.py

```diff
@@ -139,6 +139,11 @@ def parse_inputs(vehicle_deck, aircraft_values: AviaryValues(), meta_data=_MetaD
                 initial_guesses[var_name] = float(var_values[0])
                 continue
 
```

Comment:
Can you add a test for this? The test should assert that the right warning is raised. You can see an example of this here:

https://github.com/OpenMDAO/OpenMDAO/blob/c78c82526cdc2c0edd88a8d3695e6704cee0d8a1/openmdao/core/tests/test_problem.py#L1237

You won't need to actually run an optimization or anything. This is raised earlier than that.

Reply:
A unit test is added.